example/src/main/java/com/stripe/example/activity/KlarnaSourceActivity.kt

```diff
@@ -122,7 +122,6 @@ class KlarnaSourceActivity : AppCompatActivity() {
     private fun createKlarnaSource(): LiveData<Result<Source>> {
         return viewModel.createSource(
             SourceParams.createKlarna(
-                returnUrl = RETURN_URL,
```

Comment:
Seems removing this would result in `createKlarnaSource` failure, got this error in the example app: 
> Missing required param: redirect

Reply:
Did you test this with my PR?

Reply:
Reverting for this PR.